templates/product_category/_category.html.twig

```diff
@@ -0,0 +1,4 @@
+{% for categories in product_category %}
```

Comment:
{% for categories in product_category %} jest mylące, lepiej użyć 
{% for category in product_categories %} (pamiętaj żeby zaktualizować to też w kontrolerze)